### ISSUE TITLE
"Your module has been created at undefined."

### ISSUE THREAD

**Issue by tsiq-swyx** [transitive-bullshit/create-react-library#6]
nice lib so far! just trying it out and this is the output:

`create-react-library`

```bash
? Package Name react-wired
? Package Description react wired
? Author's GitHub Handle sw-yx
? GitHub Repo Path https://github.com/sw-yx/react-wired.git
? License MIT
? Package Manager npm
✔ Copying template to /Users/swyx/OpenSource/react-wired
✔ Running npm install and npm link
✔ Initializing git repo


Your module has been created at undefined.

To get started, in one tab, run:
$ cd react-wired && npm start

And in another tab, run the create-react-app devserver:
$ cd react-wired/example && npm start



Because you're using npm, you'll need to publish a dummy version of react-wired first before you can "npm link" your package into the example app.

```

this is probably an easy fix: "Your module has been created at undefined."

**Comment by transitive-bullshit** [comment]
Thanks && good catch!

This should be fixed in `v2.1.5`.

**Comment by tsiq-swyx** [comment]
thank you!
